pkg/vendir/cmd/sync.go

```diff
@@ -9,12 +9,13 @@ import (
 	"path/filepath"
 	"strings"
 
-	ctlconf "carvel.dev/vendir/pkg/vendir/config"
-	ctldir "carvel.dev/vendir/pkg/vendir/directory"
-	ctlcache "carvel.dev/vendir/pkg/vendir/fetch/cache"
 	"github.com/cppforlife/go-cli-ui/ui"
 	"github.com/mitchellh/go-homedir"
 	"github.com/spf13/cobra"
+
+	ctlconf "carvel.dev/vendir/pkg/vendir/config"
+	ctldir "carvel.dev/vendir/pkg/vendir/directory"
+	ctlcache "carvel.dev/vendir/pkg/vendir/fetch/cache"
```

Comment:
Stylewise in all the tools we do only 1 groups of imports, system and other, I think it would make sense for us to keep that consistency.

Reply:
Reverted this and in `pkg/vendir/directory/directory.go` too.

Those changes were done automatically by https://github.com/incu6us/goimports-reviser. I'll keep an eye on it next time.